hook-producer/src/handlers/webhook.rs

```diff
@@ -0,0 +1,63 @@
+use axum::{extract::State, http::StatusCode, Json};
+use hook_common::pgqueue::WebhookJobParameters;
+use url::Url;
+
+use hook_common::pgqueue::{NewJob, PgQueue};
+use serde::Serialize;
+use tracing::{debug, error};
+
+#[derive(Serialize)]
+pub struct WebhookPostResponse {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    error: Option<String>,
+}
+
+pub async fn post(
+    State(pg_queue): State<PgQueue>,
+    Json(payload): Json<WebhookJobParameters>,
```

Comment:
question: How does axum react if the json payload is malformed (i.e. cannot be deserialized).

Reply:
Looking for a way to enforce payload size limits, and we could do this by changing the deserialization to fail if size limits are exceeded.

Reply:
Yeah, I think that's good enough to get us started :+1: We own the clients, but we don't do any validation on them afaik.